recipes/htseqqc/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "v1.0" %}
+
+package:
+  name: htseqqc
+  version: {{ version }}
+
+source:
+  url: https://github.com/reneshbedre/HTSeqQC/archive/{{ version }}.tar.gz
+  sha256: 393a0e573df7420daa8f75bed52e91e65cabc908f56325ddd9a1d326115b5206
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+
+requirements:
+  host:
+    - python
+    - termcolor
```

Comment:
all those dependencies should be move to the run section, please have a look at other python based recipes